qutip/tests/test_propagator.py

```diff
@@ -127,5 +127,29 @@ def testPropHDims():
     U = propagator(H,1, unitary_mode='single')
     assert_equal(U.dims,H.dims)
 
+
+def testPropHSuperWithoutCops():
```

Comment:
Could you add another test with super `H` and using `parallel=True`.

Reply:
Added! And also added a parallel test for the case with c_ops.